src/lib/DocsSearchBar.js

```diff
@@ -158,6 +148,23 @@ class DocsSearchBar {
     }
   }
 
+  static addThemeWrapper(inputSelector, enableDarkMode) {
+    const inputElement = document.getElementById(inputSelector.substring(1))
+    const parent = inputElement.parentNode
+    const wrapper = document.createElement('div')
+
```

Comment:
```suggestion
parent.replaceChild(wrapper, inputElement)
wrapper.appendChild(inputElement)
    
```

keep dom elements manipulation together for readability and avoid loss of context